### PR TITLE
Version 5new

### PR DESCRIPTION
**Pull request by vremere-a**

### CODE REVIEW

File: src/main/webapp/web.xml [hunk 1]
```diff
@@ -3,12 +3,44 @@
...
+    <servlet-mapping>
+        <servlet-name>allUsers</servlet-name>
+        <url-pattern>/users/all</url-pattern>
+    </servlet-mapping>
+    <servlet>
+        <servlet-name>registration</servlet-name>
+        <servlet-class>com.internet.shop.controllers.RegistrationController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>registration</servlet-name>
```
Comment (RomanDubovskyi) [inline src/main/webapp/web.xml]: do we need mapping in web.xml if we use annotation for this purpose?

File: src/main/java/com/internet/shop/controllers/item/GetAllItemsController.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import com.internet.shop.service.interfaces.ProductService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/items/all")
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controllers/item/GetAllItemsController.java]:
```suggestion
@WebServlet("/products/all")
```

File: src/main/java/com/internet/shop/controllers/item/GetAllItemsController.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+public class GetAllItemsController extends HttpServlet {
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controllers/item/GetAllItemsController.java]:
```suggestion
public class GetAllProductsController extends HttpServlet {
```
fix naming consistency everywhere

File: src/main/webapp/web.xml (hunk 1 above)
Comment (vremere-a) [reply src/main/webapp/web.xml]: done

File: src/main/java/com/internet/shop/controllers/product/AddProductController.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import com.internet.shop.model.Product;
+import com.internet.shop.service.interfaces.ProductService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/product/add")
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controllers/product/AddProductController.java:13]:
```suggestion
@WebServlet("/products/add")
```
keep your naming consistent

File: src/main/java/com/internet/shop/controllers/bucket/AddItemToShoppingCartController.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+import com.internet.shop.service.interfaces.ProductService;
+import com.internet.shop.service.interfaces.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/item/add/bucket")
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controllers/bucket/AddItemToShoppingCartController.java:15]: there is example in checklist with good url naming, follow it 
https://mate-academy.github.io/jv-program-fulltime/02_jdbc_and_servlets/checklist/hw_16_1.html

**Review by RomanDubovskyi — APPROVED** [review]
not all naming was fixed, change it before merging PR

**Pull request merged**